src/utils/column-helper.ts

```diff
@@ -8,22 +9,23 @@ import { camelCase, deCamelCase, id } from '../utils';
  * @param {any[]} columns
  * @returns
  */
-export function setColumnDefaults(columns: any[]) {
+export function setColumnDefaults(columns: TableColumn[]) {
   if(!columns) return;
 
   for(let column of columns) {
     if(!column.$$id) {
       column.$$id = id();
     }
 
```

Comment:
How about just determining the valueGetter function in this code instead of the caching I did in **DataTableBodyCellComponent**?

Is it safe to assume setColumnDefaults is always run on columns?